### PR TITLE
Fix broken link of tf.train.Checkpoint.save

### PR DESCRIPTION
**Pull request by gadagashwini**

### CODE REVIEW

File: keras/engine/training.py [hunk 1]
```diff
@@ -2757,7 +2757,7 @@ def save_weights(
         While the formats are the same, do not mix `save_weights` and
         `tf.train.Checkpoint`. Checkpoints saved by `Model.save_weights` should
         be loaded using `Model.load_weights`. Checkpoints saved using
-        `tf.train.Checkpoint.save` should be restored using the corresponding
+        [tf.train.Checkpoint.save](https://www.tensorflow.org/api_docs/python/tf/train/Checkpoint#save) should be restored using the corresponding
         `tf.train.Checkpoint.restore`. Prefer `tf.train.Checkpoint` over
```
Comment (fchollet) [inline keras/engine/training.py:2761]: This one doesn't have the same issue?

File: keras/engine/training.py [hunk 2]
```diff
@@ -2757,7 +2757,7 @@ def save_weights(
         While the formats are the same, do not mix `save_weights` and
         `tf.train.Checkpoint`. Checkpoints saved by `Model.save_weights` should
         be loaded using `Model.load_weights`. Checkpoints saved using
-        `tf.train.Checkpoint.save` should be restored using the corresponding
+        [tf.train.Checkpoint.save](https://www.tensorflow.org/api_docs/python/tf/train/Checkpoint#save) should be restored using the corresponding
```
Comment (fchollet) [inline keras/engine/training.py:2760]: Please use backticks around code keywords.

**Review by fchollet — APPROVED** [review]
Thanks for the PR!

**Comment by jbischof** [comment]
@gbaned please fix the linter error so we can merge your PR. Thank you!

**Comment by gbaned** [comment]
Hi @gadagashwini  Can you please fix the PyLint errors https://github.com/keras-team/keras/runs/7870544256?check_suite_focus=true  Thank you.

File: keras/engine/training.py (hunk 1 above)
Comment (fchollet) [reply keras/engine/training.py:2762]: ?

File: keras/engine/training.py [hunk 3]
```diff
@@ -2757,7 +2757,9 @@ def save_weights(
         While the formats are the same, do not mix `save_weights` and
         `tf.train.Checkpoint`. Checkpoints saved by `Model.save_weights` should
         be loaded using `Model.load_weights`. Checkpoints saved using
-        `tf.train.Checkpoint.save` should be restored using the corresponding
+        [`tf.train.Checkpoint.save`]
+        (https://www.tensorflow.org/api_docs/python/tf/train/Checkpoint#save)
```
Comment (fchollet) [inline keras/engine/training.py:2761]: The `(` should be next to the `]`

**Comment by gbaned** [comment]
@gadagashwini Can you please check @fchollet's comments and keep us posted ? Thank you!

**Comment by fchollet** [comment]
This is fixed at head. Thanks!

**Pull request closed without merging**